app/src/components/ContributionDetail.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <template v-if="contributions.length > 0">
```

Comment:
Thoughts on removing the template and just push the if condition to section ?

Reply:
to clarify, which template (ha!), So the only hesitation that I have with moving the conditional to a `section` tag is that I believe it will show up in the DOM, where as, the `template` tag does not appear in the DOM

Reply:
> to clarify, which template

He is referring to the `template` elements that wrap each `section` element

> So the only hesitation that I have with moving the conditional to a `section` tag is that I believe it will show up in the DOM, where as, the `template` tag does not appear in the DOM

This is not entirely correct—if you implement @thelostone-mc's suggestion, each `section` tag would be part of `v-if` clause. When a `v-if` condition evaluates to false, that element does not show in the DOM. If you used a `v-show` instead, the element is in the dom, but hidden, when it evaluates to false

References with more info
- https://vuejs.org/v2/guide/conditional.html#v-if-vs-v-show
- https://stackoverflow.com/questions/51995815/vuejs-template-vs-div-or-related-for-grouping-elements-for-conditional-rend

Reply:
so the linter complains about doing this `3:5  error  This 'v-if' should be moved to the wrapper element  vue/no-use-v-if-with-v-for`  so I will keep it the way it is.

Reply:
Ah yes forgot about that. Ok great, agreed with leaving this as-is then

(More info if interested—in our case, leaving as-is would be best practice. In other cases, you'd want to instead use a computed property: https://eslint.vuejs.org/rules/no-use-v-if-with-v-for.html)